src-tauri/tauri.conf.json

```
@@ -6,7 +6,7 @@
         "distDir": "../dist"
     },
     "package": {
-        "productName": "Tari Universe",
+        "productName": "Tari Universe | Testnet",
```

```suggestion
        "productName": "Tari Universe",
```
I don't know if this will cause the application to be seen as two different applications...